app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -100,7 +100,8 @@ def update_metadata(attributes)
       end
 
       def destroy
-        unlink_from_work
+        unlink_from_work(file_set.parent)
+        unlink_rendering(file_set.parent)
```

Comment:
It looks like this combination of calls is going to save `file_set.parent` twice.  Can we make it only save once?

Reply:
I've gone back to the original single method, and disabled the two rubocop errors